pennylane/devices/default_qutrit_mixed.py

```diff
@@ -177,11 +243,13 @@ def name(self):
         return "default.qutrit.mixed"
 
     @debug_logger_init
-    def __init__(
+    def __init__(  # pylint: disable=too-many-arguments
         self,
         wires=None,
         shots=None,
         seed="global",
+        damping_measurement_gammas=None,  # PR-comment: do you prefer this or together?
+        trit_flip_measurement_probs=None,
```

Comment:
Is it better if these two inputs are one, or is it ok if they are separate like this?
Also, should I name them using measurement or readout to be consistent with default.Mixed.

Reply:
I like the idea of having just one user-facing variable that would contain both sets of values; however, I'm not sure if the parameters would still be trainable if they are input as, e.g., a dictionary. Maybe we can get the team's input on this, so that whatever is chosen also  fits well with the `qml.NoiseModel` approach (@trbromley )

Reply:
One thought I had was inputting something similar`**kwargs` instead, however since `kwargs` is just a standard naming convention it could be `**readout_probs` or something similar in our case. I will update it now and it will be easy to change back if you don't like it.

Reply:
I added the implementation, but I haven't changed the docs yet.